iceoryx_posh/test/moduletests/test_mepoo_memory_manager.cpp

```diff
@@ -30,13 +30,13 @@ using PayloadOffset_t = iox::mepoo::ChunkHeader::PayloadOffset_t;
 class MemoryManager_test : public Test
 {
   public:
-    void SetUp()
+    void SetUp() override
```

Comment:
Just a note: I guess we do not state that we override the method in most tests and while nice, it should not be mandatory for tests. For other classes we should make it a rule to mark overidden functions to benefit from additional checking of the compiler.